values-nl/strings.xml

```diff
@@ -28,27 +28,27 @@
             <string name="pref_header_install">Installatie</string>
             <!--<string name="pref_header_download">Download (noun variant, not used in English)</string>-->
             <string name="new_folder">Nieuwe map</string>
-            <string name="pref_header_gapps_selection">GApps-Selectie</string>
+            <string name="pref_header_gapps_selection">GApps-selectie</string>
             <string name="action_settings">Instellingen</string>
-            <string name="label_wifi_only">Alleen op WiFi</string>
-            <string name="label_download_dir">Downloadmap</string>
+            <string name="label_wifi_only">Alleen via wifi</string>
```

Comment:
Dat zie ik nog veel terugkomen, maar dat is niet meer correct in het Nederlands. Het is een erg ingeburgerd woord en daarom hoor je 'wifi' te schrijven. Zie https://onzetaal.nl/taaladvies/led-lamp-led-lamp-ledlamp en http://www.derozepen.nl/wifi-wifi-wi-fiwi-fi-wifi-of-wi-fi.